kerckhoff/packages/serializers.py

```diff
@@ -1,3 +1,5 @@
+import os
+from dataclasses import field
```

Comment:
I don't think you meant to import this. I'm guessing it was added by the IDE

Reply:
I used it for string manipulation.